yarn.lock

```diff
@@ -3332,7 +3332,7 @@ anymatch@~3.1.2:
     normalize-path "^3.0.0"
     picomatch "^2.0.4"
 
-"apache-arrow@>= 13.0.0", apache-arrow@^15.0.0:
+"apache-arrow@>= 15.0.0", apache-arrow@^15.0.0:
```

Comment:
Could you remove this change from this PR?

Reply:
Done